agentendpoint/agentendpoint.go

```diff
@@ -68,6 +68,7 @@ func NewClient(ctx context.Context) (*Client, error) {
 		option.WithoutAuthentication(), // Do not use oauth.
 		option.WithGRPCDialOption(grpc.WithTransportCredentials(credentials.NewTLS(nil))), // Because we disabled Auth we need to specifically enable TLS.
 		option.WithEndpoint(agentconfig.SvcEndpoint()),
+		option.WithUserAgent("Google OS Config Agent/" + agentconfig.Version()),
```

Comment:
I think user agents are not supposed to contain spaces - so `Google-OS-Config-Agent/` might be better.

Reply:
Good catch. This also doesn't affect the user agent for Cloud Log which will require a change to https://github.com/GoogleCloudPlatform/guest-logging-go as well. I'll post a PR for that too.